packages/eslint-plugin/src/rules/jsx-pragma/index.ts

```diff
@@ -144,14 +154,20 @@ export const jsxPragmaRule: Rule.RuleModule = {
     const optionsRaw = context.options[0] || {};
     const options: Options = {
       detectConflictWithOtherLibraries: optionsRaw.detectConflictWithOtherLibraries ?? true,
-      onlyRunIfImportingCompiled: optionsRaw.onlyRunIfImportingCompiled ?? false,
+      onlyRunIfImportingCompiled:
+        optionsRaw.onlyRunIfImportingCompiled ?? optionsRaw.alsoAddCompiledPragmaFor ?? false,
       runtime: optionsRaw.runtime ?? 'automatic',
+      alsoAddCompiledPragmaFor: optionsRaw.alsoAddCompiledPragmaFor ?? [],
     };
 
     const source = context.getSourceCode();
     const comments = source.getAllComments();
 
     const compiledImports = findLibraryImportDeclarations(context);
+    const otherCompiledImports = findLibraryImportDeclarations(
+      context,
+      options.alsoAddCompiledPragmaFor
```

Comment:
Can we call this `importSources`? Then it's aligned with other API usages in the Compiled compiler.